src/vcSettings.h

```diff
@@ -95,6 +99,24 @@ enum vcTileRendererFlags
   vcTRF_OnlyRequestVisibleTiles = 0x1,
 };
 
+enum vcScreenshotOutputResolution
+{
+  vcIR_720p,
+  vcIR_1080p,
+  vcIR_4k,
+  vcIR_SceneSize
+};
+
+enum class vcImageFormats
+{
+  vcIF_PNG,
+  vcIF_JPG
+};
+
+static udUInt2 ScreenshotResolutions[] = { {1280, 720}, {1920, 1080}, {4096, 2160}, {1920, 1080} };
```

Comment:
Why is the default Scene Size `1920, 1080` when the default in the settings is actually `1280, 720`?